.gitlab-ci.yml

```diff
@@ -16,8 +16,14 @@ noether-rocm:
     - export PETSC_DIR=/projects/jed/petsc PETSC_ARCH=mpich-hip-g && git -C $PETSC_DIR describe && make -C $PETSC_DIR info
 # LIBXSMM v1.16.1
     - cd .. && export XSMM_VERSION=libxsmm-1.16.1 && { [[ -d $XSMM_VERSION ]] || { git clone --depth 1 --branch 1.16.1 https://github.com/hfp/libxsmm.git $XSMM_VERSION && make -C $XSMM_VERSION -j$(nproc); }; } && git -C $XSMM_VERSION describe --tags && export XSMM_DIR=$PWD/$XSMM_VERSION && cd libCEED
-# OCCA v1.1.0
-    - cd .. && export OCCA_VERSION=occa-1.1.0 OCCA_OPENCL_ENABLED=0 && { [[ -d $OCCA_VERSION ]] || { git clone --depth 1 --branch v1.1.0 https://github.com/libocca/occa.git $OCCA_VERSION && make -C $OCCA_VERSION -j$(nproc); }; } && make -C $OCCA_VERSION info && export OCCA_DIR=$PWD/$OCCA_VERSION && cd libCEED
+# OCCA v1.1.1 (Note: Change main -> v1.1.1)
+    - >
+      cd ..
+      && export OCCA_VERSION=occa-1.1.1 OCCA_OPENCL_ENABLED=0
+      && { [[ -d $OCCA_VERSION ]] || { git clone --depth 1 --branch main https://github.com/libocca/occa.git $OCCA_VERSION && make -C $OCCA_VERSION -j$(nproc); }; }
```

Comment:
Rather than hacking `--branch main` here, I'd rather set `OCCA_VERSION` to a commit hash if this comes up in the future. (I fixed the issue manually this time.) I suppose in this instance you plan to tag v1.1.1 on the current 'main'.